server/lib/nl/common/serialize.py

```diff
@@ -106,7 +108,8 @@ def classification_to_dict(classifications: List[NLClassifier]) -> List[Dict]:
       cdict['correlation'] = True
     elif isinstance(c.attributes, QuantityClassificationAttributes):
       cdict['quantity'] = _quantity_to_dict(c.attributes)
-
+    elif isinstance(c.attributes, DateClassificationAttributes):
+      cdict['dates'] = c.attributes.dates
```

Comment:
Do we need to turn dataclass into a dict?

Reply:
done